packages/react-core/src/demos/examples/Tabs/GrayTabs.tsx

```diff
@@ -0,0 +1,158 @@
+                  />
+                  <Tab eventKey={12} title={<TabTitleText>Operators</TabTitleText>} tabContentId={`tabContent${12}`} />
+                  <Tab
+                    eventKey={13}
+                    title={<TabTitleText>Virtualization</TabTitleText>}
+                    tabContentId={`tabContent${13}`}
+                  />
+                </Tabs>
+              </FlexItem>
+              <FlexItem>
+                <TabContent
+                  key={10}
+                  eventKey={10}
+                  id={`tabContent${10}`}
+                  activeKey={activeNestedTabKey}
+                  hidden={10 !== activeNestedTabKey}
+                >
+                  <TabContentBody>
+                    {
+                      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce in odio porttitor, feugiat risus in, feugiat arcu. Nullam euismod enim eget fringilla condimentum. Maecenas tincidunt et metus id aliquet. Integer et fermentum purus. Nulla tempor velit arcu, vitae semper purus iaculis at. Sed malesuada auctor luctus. Pellentesque et leo urna. Aliquam vitae felis congue lacus mattis fringilla. Nullam et ultricies erat, sed dignissim elit. Cras mattis pulvinar aliquam. In ac est nulla. Pellentesque fermentum nibh ac sapien porta, ut congue orci aliquam. Sed nisl est, tempor eu pharetra eget, ullamcorper ut augue. Vestibulum eleifend libero eu nulla cursus lacinia.'
+                    }
+                  </TabContentBody>
+                </TabContent>
+                <TabContent
+                  key={11}
+                  eventKey={11}
+                  id={`tabContent${11}`}
+                  activeKey={activeNestedTabKey}
+                  hidden={11 !== activeNestedTabKey}
+                >
+                  <TabContentBody>Control plane panel</TabContentBody>
+                </TabContent>
+                <TabContent
+                  key={12}
+                  eventKey={12}
+                  id={`tabContent${12}`}
+                  activeKey={activeNestedTabKey}
+                  hidden={12 !== activeNestedTabKey}
+                >
+                  <TabContentBody>Operators panel</TabContentBody>
+                </TabContent>
+                <TabContent
+                  key={13}
+                  eventKey={13}
+                  id={`tabContent${13}`}
+                  activeKey={activeNestedTabKey}
+                  hidden={13 !== activeNestedTabKey}
+                >
+                  <TabContentBody>Virtualization panel</TabContentBody>
+                </TabContent>
+              </FlexItem>
+            </Flex>
+          </CardBody>
+        </Card>
+      </GridItem>
+      <GridItem xl={4} md={6}>
+        <Flex direction={{ default: 'column' }} className="pf-u-h-100">
+          <FlexItem flex={{ default: 'flex_1' }}>
+            <Card isFullHeight>
+              <CardHeader>
+                <Title headingLevel="h3">Title of Card</Title>
```

Comment:
Looking at the design, all 3 card titles are the same size, which indicates to me that they're all the same level heading, just differently sized cards. At a minimum, looks like they should all use the same font-size, but should all probably use the same element, too. wdyt @mcarrano @mmenestr @mattnolting?

Reply:
Yes, I think so

Reply:
made the change (all h3) here as well as nested tab demo

Reply:
@gitdallas can you update those to `<h2>`'s so we aren't skipping a heading level?